server/src/org/apache/cloudstack/region/RegionsApiUtil.java

```diff
@@ -186,8 +186,12 @@ protected static UserAccount makeUserAccountAPICall(Region region, String comman
                 XStream xstream = new XStream(new DomDriver());
                 xstream.alias("useraccount", UserAccountVO.class);
                 xstream.aliasField("id", UserAccountVO.class, "uuid");
-                ObjectInputStream in = xstream.createObjectInputStream(is);
-                return (UserAccountVO)in.readObject();
+                try(ObjectInputStream in = xstream.createObjectInputStream(is);) {
+                    return (UserAccountVO)in.readObject();
+                } catch (IOException e) {
+                    s_logger.error(e.getMessage());
+                    return null;
```

Comment:
this return is not needed. we will fall through to other return. You can opt not to and put another log message in there as well.